examples/websocket-proxy-server/websocket_proxy_server.metatags

```diff
@@ -0,0 +1,2 @@
+description: This example demonstrates how Ballerina acts as a proxy to another remote service via WebSocket.
```

Comment:
Edit: This BBE demonstrates how Ballerina acts as a proxy to another remote service via WebSocket.